keyboards/matrix/falcon_magnet/info.json

```
@@ -0,0 +1,90 @@
+{
+    "keyboard_name": "Falcon MS",
+    "manufacturer": "Matrix Lab",
+    "url": "",
+    "maintainer": "yulei (Astro)",
+    "usb": {
+        "vid": "0x4D53",
+        "pid": "0x474E",
+        "device_version": "1.0.0"
+    },
+    "diode_direction": "COL2ROW",
+    "indicators": {
+        "caps_lock": "A5"
+    },
+    "ws2812": {
+        "pin": "B7"
+    },
+    "processor": "STM32F411",
+    "bootloader": "stm32-dfu" ,
```

Matching community layout(s) indentified:
```suggestion
    "bootloader": "stm32-dfu" ,
    "community_layouts": ["60_hhkb"],
```